src/systems/ec/implementations/react/packages/page-header/stories/Index.jsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text } from '@storybook/addon-knobs/react';
+import Breadcrumb, { BreadcrumbItem } from '@ecl/ec-react-component-breadcrumb';
+
+import breadcrumbContent from '@ecl/ec-specs-breadcrumb/demo/data-simple';
```

Comment:
data file name should use double dash

Reply:
In our eslint config, we have a rule that forbids the use of double dash (https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/filename-case.md, defaults to kebabCase).

We have 2 exceptions:
- jsx files follow the PascalCase convention
- data files: I've added this exception in order to avoid impacting existing files. Ideally, we should rename these files that have a double dash and remove the exception.